apps/web-giddh/src/app/ledger/ledger.component.ts

```diff
@@ -997,6 +1003,13 @@ export class LedgerComponent implements OnInit, OnDestroy {
                 this.getBankTransactions();
             }
         };
+
+        this.bankMessage$.pipe(takeUntil(this.destroyed$)).subscribe(response => {
+            if (response === BANK_REFRESH_SUCCESS_MESSAGE) {
```

Comment:
change this logic show, we are getting message by API response

Reply:
Done